From: Max Mitschke <maxmitschke@fastmail.com>
Date: 2020-04-23 10:17:41 -0400
Subject: Remove duplicate typedef lua_Unsigned
References:
Upstream: N/A

This patch should only be used on the i586 build as it fails to 
build due to a duplicate typedef which also exists within the 
moonjit package. 

---
diff -Naur a/src/libraries/lua53/lstrlib.c b/src/libraries/lua53/lstrlib.c
--- a/src/libraries/lua53/lstrlib.c	2017-12-14 11:06:10.000000000 -0500
+++ b/src/libraries/lua53/lstrlib.c	2020-04-22 22:01:42.666685821 -0400
@@ -70,10 +70,6 @@
 #include "lstrlib.h"
 
 #if LUA_VERSION_NUM == 501
-typedef size_t lua_Unsigned;
-#endif
-
-#if LUA_VERSION_NUM == 501
 #	define LUAL_BUFFER53_BUFFER(B) (B)->b.buffer
 #else
 #	define LUAL_BUFFER53_BUFFER(B) (B)->b.initb
